addons/default/visiosoft/advs-module/resources/views/ad-detail/partials/content.twig

```diff
@@ -37,7 +37,9 @@
                 </div>
 
                 <!-- Content With Block -->
-                {{ addBlock('ad-detail/content',{'features':features,'adv':adv, 'profile': profile})|raw }}
+                <div id="nav-comments" class="tab-pane fade in">
```

Comment:
you can't add external module related stuff in the advs module. you can move this to the comment module